en/micro-integrator/docs/references/synapse-properties/inbound-endpoints/event-based-inbound-endpoints/rabbitmq-inbound-endpoint-properties.md

```diff
@@ -321,6 +321,9 @@ present. If set to <code>false</code>, the Micro Integrator will assume that a q
    </tbody>
 </table>
 
+!!! Note
+    The property rabbitmq.exchange.name is conditionally mandatory. If there is an existing queue with a binding, it will connect to it. And if there is no existent queue, the new rabbitmq.queue.name and rabbitmq.exchange.name are needed to create the binding.
+
```

Comment:
```suggestion
    The property rabbitmq.exchange.name becomes mandatory if you are trying to connect to a new queue, so that it can be bound to an exchange for messages to flow.

```